src/pages/Explore/index.tsx

```diff
@@ -0,0 +1,227 @@
+import NetworkFilter from 'components/Tokens/TokenTable/NetworkFilter'
+import SearchBar from 'components/Tokens/TokenTable/SearchBar'
+import TimeSelector from 'components/Tokens/TokenTable/TimeSelector'
+import TokenTable from 'components/Tokens/TokenTable/TokenTable'
+import { MouseoverTooltip } from 'components/Tooltip'
+import { useInfoExplorePageEnabled } from 'featureFlags/flags/infoExplore'
+import { useResetAtom } from 'jotai/utils'
+import { useEffect, useState } from 'react'
+import { useLocation } from 'react-router-dom'
+import styled, { css } from 'styled-components'
+import { ThemedText } from 'theme/components'
+
+const ExploreContainer = styled.div`
+  width: 100%;
+  min-width: 320px;
+  padding: 68px 12px 0px;
+
+  @media only screen and (max-width: ${({ theme }) => `${theme.breakpoint.md}px`}) {
+    padding-top: 48px;
+  }
+
+  @media only screen and (max-width: ${({ theme }) => `${theme.breakpoint.sm}px`}) {
+    padding-top: 20px;
+  }
+`
+const TitleContainer = styled.div`
+  margin-bottom: 32px;
+  max-width: ${MAX_WIDTH_MEDIA_BREAKPOINT};
+  margin-left: auto;
+  margin-right: auto;
+  display: flex;
+`
+const NavWrapper = styled.div<{ isInfoExplorePageEnabled: boolean }>`
+  display: flex;
+  max-width: ${MAX_WIDTH_MEDIA_BREAKPOINT};
+  margin: 0 auto;
+  margin-bottom: ${({ isInfoExplorePageEnabled }) => (isInfoExplorePageEnabled ? '16px' : '20px')};
+  color: ${({ theme }) => theme.neutral3};
+  flex-direction: row;
+
+  @media only screen and (max-width: ${MEDIUM_MEDIA_BREAKPOINT}) {
+    flex-direction: column;
+    gap: 8px;
+  }
+
+  ${({ isInfoExplorePageEnabled }) =>
+    isInfoExplorePageEnabled &&
+    css`
+      /* Use large media breakpoint so expanded search bar doesn't push tabs to next line */
+      @media screen and (max-width: ${LARGE_MEDIA_BREAKPOINT}) {
+        flex-direction: column;
+        gap: 16px;
+      }
+    `};
+`
+const TabBar = styled(AutoRow)`
+  gap: 24px;
+  @media screen and (max-width: ${({ theme }) => theme.breakpoint.md}px) {
+    gap: 16px;
+  }
+`
+const TabItem = styled(ThemedText.HeadlineMedium)<{ active?: boolean }>`
+  align-items: center;
+  color: ${({ theme, active }) => (active ? theme.neutral1 : theme.neutral2)};
+  cursor: pointer;
+  transition: ${({ theme }) => `${theme.transition.duration.medium} ${theme.transition.timing.ease} color`};
+`
+const FiltersContainer = styled.div<{ isInfoExplorePageEnabled: boolean }>`
+  display: flex;
+  gap: 8px;
+  height: 40px;
+
+  @media only screen and (max-width: ${MEDIUM_MEDIA_BREAKPOINT}) {
+    ${({ isInfoExplorePageEnabled }) => !isInfoExplorePageEnabled && 'order: 2;'}
+  }
+
+  @media screen and (max-width: ${({ theme }) => theme.breakpoint.md}px) {
+    ${({ isInfoExplorePageEnabled }) => isInfoExplorePageEnabled && 'justify-content: space-between;'}
+  }
+`
+const DropdownFilterContainer = styled(FiltersContainer)<{ isInfoExplorePageEnabled: boolean }>`
+  ${({ isInfoExplorePageEnabled }) =>
```

Comment:
some of the styling on this page gets hard to read bc of the small differences between Tokens & Explore designs. @cbachmeier I know you said readability will take a hit in the meantime but curious what you think on creating a separate file for Explore styled components

Reply:
you can define consts such as `newExplorePageStyles = css'...'` and then use the ternary to switch between the style objects so `${isInfoExplorePageEnabled ? newExplorePageStyles : oldExplorePageStyles` which is preferred to creating new files which would make the eventual deprecation of the new styles harder